docs/pages/api-docs/slider.md

```diff
@@ -20,77 +20,17 @@ You can learn more about the difference by [reading this guide](/guides/minimizi
 
 
 
-## Component name
 
-The `MuiSlider` name can be used for providing [default props](/customization/globals/#default-props) or [style overrides](/customization/globals/#css) at the theme level.
 
 ## Props
 
 | Name | Type | Default | Description |
 |:-----|:-----|:--------|:------------|
```

Comment:
`yarn proptypes` removed all proptypes, except the `children` prop. The question would be here whether we should use the props from the unstyled component. If we have proptypes on both we would receive twice errors for the props... Maybe we should just use the unstyled API page...